app/Transformers/UserCompactTransformer.php

```diff
@@ -70,6 +72,11 @@ public function includeCover(User $user)
         });
     }
 
+    public function includeCurrentModeRank(User $user)
+    {
+        return $this->primitive($user->statistics(Auth::user()->playmode ?? 'osu')->globalRank() ?? null);
```

Comment:
`null->x() ?? null` will still throw error.

Reply:
i tested and i believe it won't

Reply:
have you tested with someone without stats?

Reply:
yeah

Reply:
that's weird, this is what I'm seeing

![](https://s.myconan.net/2019-11/mstsc_2019-11-29_00-46-46.png)

Reply:
have you checked this again? I'm still seeing error (see above).

Reply:
been quite busy, can do today most likely

Reply:
oh yeah, you are right, i accidentally didn't change ``rank`` to ``globalRank()`` in my testing environment